test/integration/features/basic.feature

```diff
@@ -19,6 +19,16 @@ Feature: Basic
      sudoer
      """
 
+     # check the context is set as per profile
+     # In our case profile is minishift
```

Comment:
You might want to switch context as well (we can use one of the preexisting context we know does exist) and then back to _minishift_ and then run an actual command.